components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/gen/java/org/wso2/carbon/apimgt/rest/api/core/ThreatProtectionApiService.java

```diff
@@ -0,0 +1,35 @@
+package org.wso2.carbon.apimgt.rest.api.core;
+
+import org.wso2.carbon.apimgt.rest.api.core.*;
```

Comment:
Can you remove the * imports. CHeck the other places as well. there are few more.